packages/-ember-data/package.json

```diff
@@ -82,7 +82,7 @@
     "loader.js": "^4.7.0",
     "qunit": "^2.10.0",
     "qunit-dom": "^1.2.0",
-    "typescript": "~3.8.3"
+    "typescript": "~4.2.4"
```

Comment:
These should resolve to the same thing so I'm not especially worried by it, but it's slightly surprising to me to have different version requests for the top-level package vs. the nested package. I assume this was probably a case of 4.2.4 coming out while you were working on this PR, @runspired?

Reply:
indeed, fixed